tests/functional/n/namedtuple_member_inference.py

```diff
@@ -3,7 +3,6 @@
 Regression test for:
 https://bitbucket.org/logilab/pylint/issue/93/pylint-crashes-on-namedtuple-attribute
 """
-from __future__ import absolute_import, print_function
 from collections import namedtuple
 
 __revision__ = None
```

Comment:
This is in a lot of file and I think it's useless too. Would you mind also removing them in this cleanup MR ?
```suggestion
```

Reply:
Sure! There's a good few of these also